recipes-graphics/wayland/weston-init/weston.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Weston Wayland Compositor
+RequiresMountsFor=/run
+
+[Service]
+User=root
+EnvironmentFile=-/etc/default/weston
+ExecStartPre=/usr/bin/weston-waitforcard.sh
```

Comment:
I'm not a systemd guru, there is a already existing method in systemd for waiting a path exists and start/start the service based on that condition:

PathExists=

Documentation:
https://www.freedesktop.org/software/systemd/man/systemd.path.html

I don't the current patch is the right approach for this problem.